File: CS/CustomItemsSample/CustomItems/SunburstChart/SunburstItemMetadata.cs
using DevExpress.DashboardCommon;
using System.ComponentModel;

namespace CustomItemsSample {
    [
    DisplayName("Sunburst"),
    CustomItemDescription("Sunburst description"),
    CustomItemImage("DashboardMainDemo.Images.SunburstCustomItem.svg")
    ]
    public class SunburstItemMetadata : CustomItemMetadata {
        readonly DimensionCollection arguments = new DimensionCollection();
        [
        DisplayName("Value"),
        EmptyDataItemPlaceholder("Value"),
        ]
        public Measure Value {
            get { return GetPropertyValue<Measure>(); }
            set { SetPropertyValue(value); }
        }
        [
        DisplayName("Arguments"),
        EmptyDataItemPlaceholder("Argument"),
        SupportColoring(DefaultColoringMode.None),
        SupportInteractivity
        ]
        public DimensionCollection Arguments { get { return arguments; } }
    }
}

File: VB/CustomItemsSample/CustomItems/SunburstChart/SunburstItemMetadata.vb
Imports DevExpress.DashboardCommon
Imports System.ComponentModel

Namespace CustomItemsSample
	<DisplayName("Sunburst"), CustomItemDescription("Sunburst description"), CustomItemImage("CustomItemsSample.Images.SunburstCustomItem.svg")>
	Public Class SunburstItemMetadata
		Inherits CustomItemMetadata

'INSTANT VB NOTE: The field arguments was renamed since Visual Basic does not allow fields to have the same name as other class members:
		Private ReadOnly arguments_Conflict As New DimensionCollection()
		<DisplayName("Value"), EmptyDataItemPlaceholder("Value"), >
		Public Property Value() As Measure
			Get
				Return GetPropertyValue(Of Measure)()
			End Get
			Set(ByVal value As Measure)
				SetPropertyValue(value)
			End Set
		End Property
		<DisplayName("Arguments"), EmptyDataItemPlaceholder("Argument"), SupportColoring(DefaultColoringMode.None), SupportInteractivity>
		Public ReadOnly Property Arguments() As DimensionCollection
			Get
				Return arguments_Conflict
			End Get
		End Property
	End Class
End Namespace
